src/Microsoft.Sbom.Api/FormatValidator/IValidatedSBOM.cs

```diff
@@ -11,4 +11,6 @@ public interface IValidatedSBOM
     public Task<FormatValidationResults> GetValidationResults();
 
     public Task<FormatEnforcedSPDX2> GetRawSPDXDocument();
+
+    public void Dispose();
```

Comment:
Suggestion: Rather than adding the `Dispose` method explicitly, derive `IValidatedSBOM` from `IDisposable`. That buys you the syntactic sugar of `using` blocks

Reply:
Done!